docs/setup/independent/install-kubeadm.md

```diff
@@ -20,6 +20,16 @@ This page shows how to use install kubeadm.
 
 {% capture steps %}
 
+## Verify the MAC address and product_uuid are unique for every node
+
+* You can get the MAC address of the network interfaces using the command `ip link` or `ifconfig -a`
+* The product_uuid can be checked by using the command `sudo cat /sys/class/dmi/id/product_uuid`
```

Comment:
This seems to be the case in Debian and Ubuntu, but I'm wondering if there are other mainstream distros which don't use this...